web/src/components/Header.tsx

```diff
@@ -140,8 +141,11 @@ export const Header = ({
           </ActionBar>
         </Row>
         {hasNavigationBar && (
-          <NavigationBarScrollContainer scrollContainerRef={scrollContainerRef} direction={direction}>
-            <NavigationBar>{navigationItems}</NavigationBar>
+          <NavigationBarScrollContainer
+            scrollContainerRef={scrollContainerRef}
+            direction={direction}
+            activeIndex={navigationItems.findIndex(el => el.props.active)}>
+            <NavigationBar id='navbar'>{navigationItems}</NavigationBar>
```

Comment:
```suggestion
            <NavigationBar id='navigation-bar'>{navigationItems}</NavigationBar>
```
or, depending on conventions
```suggestion
            <NavigationBar id='navigationBar'>{navigationItems}</NavigationBar>
```